AssetTracker/apps/functionaltests/test_asset.py

```diff
@@ -5,29 +5,29 @@
 from selenium.webdriver.support import expected_conditions as EC
 from django.test import LiveServerTestCase
 from django.contrib.auth.models import User
-from assets.models import Asset
-from companies.models import Company
-from companyusers.models import CompanyUser
+from ..assets.models import Asset
+from ..companies.models import Company
+from ..companyusers.models import CompanyUser
 import time
 
 class AssetTest(LiveServerTestCase):
 
   def setUp(self):
     self.browser = webdriver.Firefox()
-    self.company = Company(Name="Makers")
+    self.company = Company(name="Makers")
     self.company.save()
     self.user = User.objects.create_user(username='admin1', password='admin1', email='test@test.com', is_active=True)
     self.user.save()
-    self.company_user = CompanyUser.objects.create(User=self.user, Company=self.company)
-    self.company2 = Company(Name="IBM")
+    self.company_user = CompanyUser.objects.create(user=self.user, company=self.company)
+    self.company2 = Company(name="IBM")
     self.company2.save()
     self.user2 = User.objects.create_user(username='admin2', password='admin2', email='test2@test.com', is_active=True)
     self.user2.save()
-    self.company_user2 = CompanyUser.objects.create(User=self.user2, Company=self.company2)
+    self.company_user2 = CompanyUser.objects.create(user=self.user2, company=self.company2)
     self.user3 = User.objects.create_user(username='admin3', password='admin3', email='test3@test.com', is_active=True)
     self.user3.save()
-    self.company_user3 = CompanyUser.objects.create(User=self.user3, Company=self.company)
-    self.A = Asset(AssetTag='BR20RL', DeviceType='Laptop', AssetStatus='Ready', SerialNumber='5', CreatedBy=self.user, Company=self.company)
+    self.company_user3 = CompanyUser.objects.create(user=self.user3, company=self.company)
```

Comment:
Space before and after '=' signs

Reply:
I am a bit skeptical about this, I tried and it looks much worse 🤣